packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1481,6 +1630,9 @@ class TextSelectionGestureDetectorBuilder {
     // trigger the selection overlay.
     // For backwards-compatibility, we treat a null kind the same as touch.
     final PointerDeviceKind? kind = details.kind;
+    // TODO(justinmc): Should a desktop platform show its selection toolbar when
+    // receiving a tap event?  Say a Windows device with a touchscreen.
```

Comment:
Great question there - and I think it definitely should. Just added comments in #106586 and opened another PR to enable selection handles.

Reply:
Thanks, I'm looking forward to straightening that out in Flutter.  I keep running into it.